docs/visual-testing/mobile-native-testing.md

```diff
@@ -25,6 +25,19 @@ When writing a visual test for mobile apps, we recommend the following:
 ## Limitations
 
 The following features are not yet available for mobile app testing:
-- Full page screenshots
-- DOM capture and inspection
 - [Selective diffing](./selective-diffing.md)
+
+Native full-page screenshots are currently in beta and may have unexpected behavior.
+Identified limitations:
+- Slow screenshot capture
+- 1 pixel shifts in the screenshot (iOS tablets only)
+- Ignoring and clipping limited to elements within `scrollElement` (iOS only)
+- Sticky elements missing at the bottom of the screen
+- The DOM capturing, inspection and selective diffing features have not yet been implemented
```

Comment:
This seems redundant with https://github.com/saucelabs/sauce-docs/pull/3025/files#diff-73428f3c04a010791939b5273874e6c02419be93c2acb1a532d2e7f9fccda5ceR30

Reply:
FPS DOM is not captured

Reply:
I'd probably write it that way then:

```suggestion
- DOM capture is not available
```

Reply:
updated